addons/l10n_es_edi_facturae/wizard/account_move_reversal.py

```diff
@@ -5,30 +5,10 @@ class AccountMoveReversal(models.TransientModel):
     _inherit = 'account.move.reversal'
 
     l10n_es_edi_facturae_reason_code = fields.Selection(
-        selection=[
```

Comment:
Put it in a generic variable then and import it

Reply:
Because it is done like that elsewhere and you see where it comes from instead of obscurely stealing it from another field through the orm inspection.  REASON_CODE_SELECTION

Reply:
> Because it is done like that elsewhere

It is done wrong in many places.
For instance, do you know how to handle translations correctly?

Reply:
It is also easier to read: on the model where it matters (`account.move`), we can see all the options available directly without needing to go elswhere.
On models where it doesn't (here), we just copy because it is what it is in practice: only a proxy

Reply:
In the original module they will be in the translations and I am not aware of any problem in the case of an import.